types/algoliasearch/index.d.ts

```diff
@@ -1650,6 +1650,12 @@ declare namespace algoliasearch {
      * such as ignorePlurals, removeStopWords, and CJK word-detection.
      */
     queryLanguages?: Array<'af' | 'ar' | 'az' | 'bg' | 'bn' | 'ca' | 'cs' | 'cy' | 'da' | 'de' | 'el' | 'en' | 'eo' | 'es' | 'et' | 'eu' | 'fa' | 'fi' | 'fo' | 'fr' | 'ga' | 'gl' | 'he' | 'hi' | 'hu' | 'hy' | 'id' | 'is' | 'it' | 'ja' | 'ka' | 'kk' | 'ko' | 'ku' | 'ky' | 'lt' | 'lv' | 'mi' | 'mn' | 'mr' | 'ms' | 'mt' | 'nb' | 'nl' | 'no' | 'ns' | 'pl' | 'ps' | 'pt' | 'pt-br' | 'qu' | 'ro' | 'ru' | 'sk' | 'sq' | 'sv' | 'sw' | 'ta' | 'te' | 'th' | 'tl' | 'tn' | 'tr' | 'tt' | 'uk' | 'ur' | 'uz' | 'zh'>;
+    /**
+     * Set the maximum number of hits accessible via pagination.
+     * We set the max number of *hits*, not max number of pages.
+     * Works with the page and hitsByPage settings to establish the full paging logic.
```

Comment:
```suggestion
     * Works with the page and hitsByPage settings to establish the full paging logic.
     * https://www.algolia.com/doc/api-reference/api-parameters/paginationLimitedTo/?language=javascript
```

Reply:
I didn't include the URL because last time someone removed them all:
https://github.com/DefinitelyTyped/DefinitelyTyped/commit/4a8c51aae8ebfe299d7a13e98848039d148576ce#diff-a7ffbfb783e3a1e7813277f5d2711c13

Reply:
oh that's right, more consistent to leave it out then I guess? I wouldn't say it matters much for now

Reply:
I prefer with URLs, I can even add URLs for the other options if you'd like ;)